### PR TITLE
Graphql two

### PR DESCRIPTION
**Pull request by catlynbowles**
** Describe the functionality to be tested: **

(Bullet list feature/functionality)
- Install apollo and graphql
- Import Apollo provider to index & create new instance of Client
- Basic query of policies
- Basic query of misinformation clinics & authorized clinics

** Giphy of how you feel submitting this PR: **

https://giphy.com/clips/gerbert-love-fun-puppet-M54VH39DQ3e7lhWKvQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/client": "^3.6.9",
```
Comment (Rallen13) [inline package.json:6]: Cool! I still need to educate myself on Apollo more.

File: src/Components/NavBar/NavBar.js [hunk 2]
```diff
@@ -2,23 +2,26 @@ import React from "react";
...
-              <h1>STATE STATUS</h1>
-            </div>
           </div>
         </Link>
-      </Routes>
+            <div className="nav-tabs">
+              <Link to="/about" className="nav-tabs">
+                <h1>ABOUT</h1>
+              </Link>
+              <Link to="/clinics" className="nav-tabs">
```
Comment (lkarpins) [inline src/Components/NavBar/NavBar.js:18]: Nice work gettin those Links working! 🔗

File: src/Views/ClinicsView/ClinicsView.js [hunk 3]
```diff
@@ -1,23 +1,54 @@
...
 
-  // useEffect(() => {
-  //     getAllResources().then((data) => setResources(data.data));
-  //     getAllAuthorizedClinics().then((data) => setAuthorizedClinics(data.data));
-  //     getAllMisinformationClinics().then((data) =>
-  //       setMisinformationCenters(data.data)
-  //     );
-  //   }, []);
+const GET_CLINICS = gql`
+  query {
```
Comment (Rallen13) [inline src/Views/ClinicsView/ClinicsView.js:6]: This looks straight forward to query what you need.

File: package.json (hunk 1 above)
Comment (lkarpins) [inline package.json:6]: Glad we figured out the version issue!

File: src/index.js [hunk 4]
```diff
@@ -4,14 +4,24 @@ import "./index.scss";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
 import { BrowserRouter } from "react-router-dom";
+import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
```
Comment (Rallen13) [inline src/index.js:7]: So this is how you import ApolloClient.

File: src/Views/ClinicsView/ClinicsView.js [hunk 5]
```diff
@@ -1,23 +1,54 @@
...
-  //   const [resources, setResources] = useState([]);
 
-  // useEffect(() => {
-  //     getAllResources().then((data) => setResources(data.data));
-  //     getAllAuthorizedClinics().then((data) => setAuthorizedClinics(data.data));
-  //     getAllMisinformationClinics().then((data) =>
-  //       setMisinformationCenters(data.data)
-  //     );
-  //   }, []);
+const GET_CLINICS = gql`
```
Comment (lkarpins) [inline src/Views/ClinicsView/ClinicsView.js:5]: Nice job with querying the data, Cat!

**Review by lkarpins — APPROVED** [review]

**Review by Rallen13 — COMMENTED** [review]
This looks awesome! Thanks so much for doing this guys!

File: src/index.js [hunk 6]
```diff
@@ -4,14 +4,24 @@ import "./index.scss";
...
+
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
+
+const client = new ApolloClient({
```
Comment (irmakerdem) [inline src/index.js:12]: Nice job keeping this in index instead of it's own file!

**Review by irmakerdem — APPROVED** [review]
Looks great! FE & BE are talking thanks to gql!

**Pull request merged**